docs/src/pages/[platform]/connected-components/storage/fileuploader/examples/FileUploaderComponentsExample.tsx

```diff
@@ -5,22 +5,22 @@ export const FileUploaderComponentsExample = () => {
     <FileUploader
       variation="drop"
       acceptedFileTypes={['image/*']}
-      level="public"
-      components={{
-        UploadDropZone: ({ inDropZone, children, ...rest }) => {
-          return (
-            <View
-              padding="xl"
-              backgroundColor={
-                inDropZone ? 'brand.secondary.40' : 'brand.secondary.20'
-              }
-              {...rest}
-            >
-              <Text textAlign="center">Drop files here</Text>
-            </View>
-          );
-        },
-      }}
+      storageLevel="public"
+      // components={{
+      //   UploadDropZone: ({ inDropZone, children, ...rest }) => {
+      //     return (
+      //       <View
+      //         padding="xl"
+      //         backgroundColor={
+      //           inDropZone ? 'brand.secondary.40' : 'brand.secondary.20'
+      //         }
+      //         {...rest}
+      //       >
+      //         <Text textAlign="center">Drop files here</Text>
+      //       </View>
+      //     );
+      //   },
+      // }}
```

Comment:
This needs a bit more context either as comment or PR description, we normally don't submit commented out code for review.